README.md

```diff
@@ -45,6 +45,12 @@ User groups with pre defined roles can be creating by providing the existing rol
 2. CDK deploy change and note `veda-auth-stack-<STAGE>.userpoolid` in output. It will include the deployment region and a UUID, for example `us-west-2:11111111-1111-1111-1111-111111111111`
 3. Add a new statement to the role's trust policy in the AWS IAM console. Navigate to the desired role, choose `Trust Relationship` and select `edit`--be careful to preserve the existing trust statements when appending a new statement for this identity pool.
 
+## Creating a data access role
```

Comment:
Thanks for adding documentation. In the case of adding the data access role, I was wondering how you operated things, i.e, what you actually _did_ on your terminal to create that role. Did you just follow the steps you outlined below and then ran `cdk deploy` ?

Reply:
Right, the same `cdk deploy --profile [profile-name]` command after setting up the environment configuration file. Sorry if I misinterpreted your previous question about documentation

Reply:
Nope all good both answers were helpful